### PR TITLE
[ci] Update destination dir on s3

### PR DESCRIPTION
**Pull request by alexandrebouthinon**
## What does this PR do ?
Update destination dir on s3

### How should this be manually tested?
You cannot :/

### Other changes
Wait 10 sec to be sure all lambda execution have been completed.

### CODE REVIEW

**Comment by Aschen** [comment]
The documentation test framework has to be updated accordingly otherwise it can't be able to download the SDK to run the tests

File: .travis.yml [hunk 1]
```diff
@@ -103,30 +81,18 @@ jobs:
...
         region: us-west-2
         skip_cleanup: true
-        upload-dir: sdk/cpp/$TRAVIS_BRANCH
+        upload-dir: releases/sdk/cpp/$TRAVIS_BRANCH
         local-dir: deploy
         on:
           all_branches: true
           condition: $TRAVIS_BRANCH =~ ^master|.?-dev$
 
-      after_deploy:
```
Comment (Aschen) [inline .travis.yml]: I'm not quite sure to understand, if you remove the invalidation, the SDK served by cloudfront will stay the same no ?

Comment (alexandrebouthinon) [reply .travis.yml]: No the cloudfront invalidation will be handle by dl.kuzzle.io lamda. So we do not need to spam cloudfront with a second invalidation.

**Comment by alexandrebouthinon** [comment]
Useless now

**Pull request closed without merging**